docs/src/main/tut/typeclasses/lawtesting.md

```diff
@@ -0,0 +1,146 @@
+# Law testing
+
+Laws are an important part of cats.
+They help us enforce constraints that we usually assume to be true and give us guarantees that our code does what we think it does.
+Manually testing each data type to adhere to each of its type class instances can be quite cumbersome.
+Thankfully cats comes with the `cats-testkit`, which makes it a lot easier to write tests for type class instances using laws.
```

Comment:
well, strictly speaking, `catalysts` and `discipline` are the two libraries to make it easier to "write tests for type class instances using laws.", `cats-testkit` is just a tiny lib on top of them to make it slightly more convenient to test against cats type classes using scalatests.  relevant discussion https://gitter.im/typelevel/sbt-catalysts?at=59a85afdee5c9a4c5f1d45fb

Reply:
That's true, what do you think of rewording to 
"Thankfully cats comes with the `cats-testkit`, which is based on `catalysts` and `discipline`, both of which make it a lot easier to write law tests for type class instances."
? :)

Reply:
I think, it's probably more accurate to break it into two sentences. 
catalysts and discipline are the two libraries to make it easier to "write tests for type class instances using laws. cats-testkit makes it more convenient to test against cats type classes using scalatests. paging @BennyHill for better wording.